classifiers/nlp/ZeroShotTFClassifier/requirements.txt

```
@@ -1,2 +1,3 @@
+jina==1.0.15
```

Do we always need to specify the jina version in the requirements? `jina build` will automatically add the version information.

```suggestion
jina==1.0.15
```

---

To enable version bumps, jina version must be mentioned either in `Dockerfile` or `requirements.txt`. If we don't pass a version number, there won't be any "version bump" PRs. As we are reworking on cookiecutter, during executor creation, we can fill in the version number too.